NixOS/nixpkgs
# cargo-bolero: init at 0.6.2

THIS IS NOT YET READY TO MERGE: I haven't managed yet to actually fuzz with it so there may still be something wrong with it. That said it already works for generating code

### nixpkgs-check report

**version:** `nixpkgs-check v0.1.0` on NixOS 21.11, sandbox = "true"
**packages declared changed:** {"cargo-bolero"}
**manual tests declared performed:**
 * ✔ built on NixOS
 * 😢 not built on MacOS
 * 😢 not built on Other Linux distributions
 * generated a test case to verify things seem to build

**complies with contributing.md:** ✔ yes
**package cargo-bolero:** 💚 started building again
**tests of cargo-bolero:** 😢 there are no tests
